app/controllers/signup/payments_controller.rb

```diff
@@ -18,10 +18,14 @@ def create
 
       result = checkout.checkout_url(amount: @form.amount, email: @member.email, return_to: callback_signup_payments_url, member_id: @member.id, date: Date.current)
       if result.success?
+        if Rails.env.development?
+          # It takes the sandbox a while to be ready
+          sleep 2
+        end
```

Comment:
This is nasty, but I couldn't come up with a better way to make testing against the sandbox work more than 1/10 of the time.

Reply:
🍳

Reply:
I'm also realizing that this won't handle staging correctly, so I guess I'll make this an env var that we can set in each environment.